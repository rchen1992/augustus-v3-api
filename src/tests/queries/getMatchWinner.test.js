const mockMatchesRepo = require('@tests/mockRepos/match');
const { testQuery } = require('@tests/utils');

testQuery({
    id: "Get a match's winner",
    query: `
        {
            match(id: 1) {
                id
                winner {
                    id
                    userName
                }
            }
        }
    `,
    variables: {},
    context: () => {
        return {
            repos: {
                match: mockMatchesRepo,
            },
        };
    },
    expected: undefined,
});
