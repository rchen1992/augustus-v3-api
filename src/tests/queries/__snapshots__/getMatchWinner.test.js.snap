// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`query: Get a match's winner 1`] = `
Object {
  "data": Object {
    "match": Object {
      "id": "1",
      "winner": Object {
        "id": "1",
        "userName": "cblanko0",
      },
    },
  },
}
`;
